EIPS/eip-7776.md

```diff
@@ -0,0 +1,192 @@
+---
+eip: 7776
```

Comment:
```suggestion
eip: 7778
```
Assigning next sequential EIP/ERC/RIP number.  7776 was already assigned to an ERC

Please also update the filename.

Reply:
Renamed filename